spec/factories/kudos.rb

```diff
@@ -4,5 +4,6 @@
     sequence(:content) { |n| "factory bot test content #{n}" }
     giver { build :employee }
     receiver { create :employee }
+    company_value { create :company_value }
```

Comment:
FActoryBot can also guess the name of the class :) 
```suggestion
    company_value
```